donation/views.py

```diff
@@ -16,15 +17,26 @@ def default(self, obj):
 
 # Create your views here.
 def donation_create(request):
+    if request.user.is_anonymous:
+        form= CreateNewDonation()
+    else:
+        form = CreateNewDonation(initial={'ong': request.user.ong})
     if request.method == "POST":
         form = CreateNewDonation(request.POST)
         if form.is_valid():
+            if not request.user.is_anonymous:
+                ong=request.user.ong
+                donation=form.save()
```

Comment:
If you use what I told you about login required decorator, change this line to "donation = form.save(commit=False)"